anomalib/models/dfkde/config.yaml

```diff
@@ -50,6 +50,8 @@ logging:
   logger: [] # options: [tensorboard, wandb, csv] or combinations.
   log_graph: false # Logs the model graph to respective logger.
 
+optimization:
+  export_mode: ""
```

Comment:
samet as above, and for the rest of the configuration files.

Reply:
addressed in: [83d0286](https://github.com/openvinotoolkit/anomalib/pull/509/commits/83d02868fbe31e9b4246a66fac33de1687d4ed5a)